tests/CUtilHelloWorld/c/main

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
This could be the basis of an automated test.

Reply:
Absolutely, so right now I am planning to add a "make-all" in the root of tests and just build and execute them, but (as always) I am open to better ideas :) - In an ideal world this should be always integrated with the ROS Building Service, so we get a report of what passes with the Pull Request, can you please look into this?

Reply:
You could use an assert-like macro which, if its argument evaluates to false, essentially does
````
*SetEval Test$Passed 0
*SetEval Test$FailLine 34
*Set Test$FailFile "main.c"
*Set Test$FailName "atoi_BadInput"
*Error "Test Failed"
````
and then the calling program can record/report that failure.